go/vt/vttablet/tabletserver/schema/engine_test.go

```diff
@@ -95,6 +96,19 @@ func TestOpenAndReload(t *testing.T) {
 	assert.Equal(t, int64(0), se.tableFileSizeGauge.Counts()["msg"])
 	assert.Equal(t, int64(0), se.tableAllocatedSizeGauge.Counts()["msg"])
 
+	t.Run("EnsureConnectionAndDB", func(t *testing.T) {
```

Comment:
The queries that they run aren't added in the `fakemysqldaemon`. So, if they were to run the schema change detection the test would fail. (I tested that it failed without the changes to prevent schema change detection)